### DISCUSSION TITLE
Java Hand Animation for Bedrock/Pocket Edition

### DISCUSSION THREAD

**Discussion by Jus-fly-to-saturn** [BedrockTweaks/Files#349] [Ideas]
I would like to suggest to create resources pack called **Java Hand Animation** similar to exact version of Minecraft java to Minecraft Bedrock or Pocket Edition of the game, not only for hitting, I would also like to suggest that when you do this, it is connected to java, just like using a bow and sword, java combats are smoother than bedrock, that's why I would like to suggest this because the Java hands i downloaded is not the exactly match to java not like to Java animation v2, but the Java animation v2, the capes are disabled and I don't like it..

Thanks for reading this!

Have a great day^^

**Comment by SquatchHunter** [comment]
not sure if entirely possible due to the engine being so vastly different.
